apps/mobile/src/components/home/home-with-account-screen.tsx

```diff
@@ -101,7 +104,7 @@ export function HomeScreenWithAccount({ currentAccount }: HomeScreenWithAccountP
                     ? () =>
                         onOpenToken?.({
                           assetProtocol: CryptoAssetProtocols.nativeBtc,
-                          assetId: 'BTC',
+                          assetId: serializeAssetId(getAssetId(btcAsset)),
```

Comment:
@alexp3y : `BTC` / `STX` don't need `assetId` but I am doing it for route consistency.

Now that I am serialising the `assetId` and it contains the protocol, perhaps I should revisit the route structure and remove the protocol level? `'/(tabs)/(index)/[assetProtocol]/[assetId]'` => `'/(tabs)/(index)/[assetId]'`

Components can still be grouped in folders but maybe this will be a better approach to use in the extension also as we can have cleaner visible URLs?